OpenSim/Actuators/Millard2012AccelerationMuscle.cpp

```diff
@@ -226,10 +226,6 @@ Millard2012AccelerationMuscle::Millard2012AccelerationMuscle()
 {    
     setNull();
     constructProperties();
-
-    // TODO: Remove this once MuscleFirstOrderActivationDynamicModel and
-    //       MuscleFixedWidthPennationModel have been made into properties.
-    finalizeFromProperties();
 }
```

Comment:
Is this related somehow (the removal of finalizeFromProperties)?

Reply:
Yes.
1. Components should not call `finalizeFromProperties` in their constructors (except for Model, which we agreed that convenience was worth the price of some inconsistency). This should have been removed earlier.
2.  Keeping it in causes the (invalid) muscle (has no path points) to be finalized which throws an exception that there are no path points, which this PR added.